### PR TITLE
Make it prettier.

### PR DESCRIPTION
**Pull request by Osmose**
The CSS and HTML are only kinda okay, but it works!

### CODE REVIEW

**Comment by Osmose** [comment]
Live at http://mozilla.github.io/webdev/ if you want to play around with it.

File: src/css/webdev.css [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+    --project-bg: linear-gradient(var(--cloud-blue), var(--light-grey));
+}
+
+* {
+    box-sizing: border-box;
+}
+
+body {
+    background: #1E1E21  no-repeat scroll center top / 100% 500px;
```
Comment (mythmon) [inline src/css/webdev.css]: Double space.

File: src/index.html [hunk 2]
```diff
@@ -1,20 +1,54 @@
...
+{% from 'macros.lib.html' import project_box %}
 
 <!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="utf-8">
     <title>Mozilla Webdev</title>
+    <link rel="stylesheet" href="//code.cdn.mozilla.net/fonts/fira.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/3.0.2/normalize.min.css">
+    <link href="//mozorg.cdn.mozilla.net/media/css/tabzilla-min.css" rel="stylesheet">
```
Comment (koddsson) [inline src/index.html]: Is there a reason why this is protocol-less? Accessing the asset on http `will` redirect to `https` so I think it's cool to set this as explicit `https`. (and probably the same for all the other protocol-less urls).

>[2014.12.17: Now that SSL is encouraged for everyone and doesn’t have performance concerns, this technique is now an anti-pattern. If the asset you need is available on SSL, then always use the https:// asset.](http://www.paulirish.com/2010/the-protocol-relative-url/)

**Comment by koddsson** [comment]
I built this branch locally and the CSS compilation is failing because of these too undefined variables:

`--bg-gradient` and `--project-bg`

File: src/index.html (hunk 2 above)
Comment (Osmose) [inline src/index.html]: Because the tabzilla docs that I copy-pasted it from were protocol-less, of course. :D

Easy to fix though.

Comment (koddsson) [inline src/index.html]: :P

**Comment by Osmose** [comment]
> I built this branch locally and the CSS compilation is failing because of these too undefined variables:
> 
> --bg-gradient and --project-bg

I fixed those but pushed to my fork instead of this particular branch, sorry!

(Why didn't I make this PR from my fork in the first place? I somehow got a second fork created called webdev-1 via `hub fork`, then deleted my original repo, then Github refused to let me see my webdev-1 fork until I was able to delete it and then recreate a new webdev fork on my account. While it was still broken I decided to just push to origin and submit the PR.)

File: src/css/webdev.css [hunk 3]
```diff
@@ -0,0 +1,139 @@
+:root {
+    --text-color: #FFF;
+    --link-color: #0095DD;
+    --background: radial-gradient(ellipse farthest-side at left top,
+                                   #00549E 0px, transparent 100%)
+                   #1E1E21;
```
Comment (mythmon) [inline src/css/webdev.css]: Nit: I suspect you wanted this one space to the left.

File: src/css/webdev.css [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+    padding: 10px 0 5px;
+}
+
+.footer-logo {
+    /* Retina */
+    height: calc(83px / 2);
+    width: calc(260px / 2);
+}
+
+
```
Comment (mythmon) [inline src/css/webdev.css]: Nit: too many blank lines at the end.

File: src/css/webdev.css [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+}
+
+#masthead h1 {
+    margin: 0;
+}
+
+.wordmark {
+    /* Retina */
+    height: calc(168px / 2);
+    width: calc(311px / 2);
```
Comment (mythmon) [inline src/css/webdev.css]: The comment here could be clearer.

**Comment by mythmon** [comment]
This looks good. There are a couple of nits, but you should ship it with or without fixing them. :boat: 

r+

**Pull request merged**